cmd/sriov/main.go

```diff
@@ -74,7 +74,24 @@ func cmdAdd(args *skel.CmdArgs) error {
 	defer netns.Close()
 
 	sm := sriov.NewSriovManager()
-	if err := sm.ApplyVFConfig(netConf); err != nil {
+	pfLink, err := sm.FillOriginalVfInfo(netConf)
+	if err != nil {
+		return fmt.Errorf("failed to lookup master %q: %v", netConf.Master, err)
+	}
+	defer func() {
+		if err != nil {
+			err := netns.Do(func(_ ns.NetNS) error {
+				_, err := netlink.LinkByName(args.IfName)
+				return err
+			})
+			if err == nil {
+				_ = sm.ReleaseVF(netConf, args.IfName, args.ContainerID, netns)
+				// Reset the VF if failure occurs before the netconf is cached
+				_ = sm.ResetVFConfig(netConf)
```

Comment:
we should always do this IMO, regardless of the fact that netdev is present in container namespace.
currently if ApplyVFConfig fails it will not call ResetVFConfig here.

so lets call it after L91